pages/common/git-repl.md

```diff
@@ -0,0 +1,21 @@
+# git repl
+
+> Git REPL (read-evaluate-print-loop) - an interactive Git shell.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-repl>.
+
+- Start an interactive Git shell:
+
+`git repl`
+
+- Run a Git command while in the interactive Git shell:
+
+`{{git_subcommand}} {{command_arguments}}`
+
+- Run an external (non-Git) command while in the interactive Git shell:
+
+`!{{command}}`
```

Comment:
```suggestion
`!{{command}} {{command_arguments}}`
```

Reply:
I'd assume the user would get that

Reply:
I think it's better to be explicit here, although the command does not actually need arguments.

Reply:
should I commit then...

Reply:
If in doubt, it's always better to explicit @CleanMachine1. tldr pages are aimed at a wide audience, but we must especially keep beginners in mind here.

Reply:
It just makes it seem you need arguments, however I will agree, that it might help a beginner